select/transactions.md

```diff
@@ -83,7 +85,9 @@ fileName:transaction.json
   },
   "card": {
     "id": "bc538b71-31c5-4699-820a-6d4a08693314",
+    "firstNumbers": "555500",
```

Comment:
nitpicking: visa cards always starts with 4 more about this [here](https://www.bankrate.com/credit-cards/advice/what-do-the-numbers-on-your-credit-card-mean/)

Reply:
Corrected ➡️ [4e0e6bb](https://github.com/Enigmatic-Smile/docs/pull/267/commits/4e0e6bb55d40f0a692b4e3369b7c5d81d8a421d5)